code/style.css

```diff
@@ -138,3 +148,28 @@ button:hover {
   opacity: 0.9;
   transition: all 0.2s ease;
 }
+
+select {
+  background-color: #393E46;
+  color: white;
+  border-radius: 4px;
+  padding: 16px 40px;
+  margin-right: 4px;
+  font-size: 16px;
+  line-height: 26px;
+  font-family: 'Montserrat';
+  font-weight: 500;
+  cursor: pointer;
+}
+
+option {
+  font-size: 16px;
+  line-height: 26px;
+  font-family: 'Montserrat';
+  font-weight: 500;
+}
+
+.thankyou {
+  color: #F5EDED;
+  text-align: center;
+}
```

Comment:
Code review Jenny & Christoffer:
we started with looking at bot and try all the functions. Love the chatbot/user images/symbols J And the styling overall looks great.
On one of the buttons and in the scroll menu I can´t see the pokebowl-symbol? (maria is using a Mac, don´t know if that has anything to do with it but sometimes it´s just that)
It was a good flow and the time-funtion is working all the way to the end. Really liked the end page with “Thank you for the order”, our only suggestion would be to shorten the timout setting.

Code is very clean, with good structure and we like all the comments, explaining what the different functions is for(or do). Maria specially liked the /*interaction comments wich made it really easy to follow in the bot conversation code. Jessica really liked the design!!
It all looks semantic and in line with Technigos code-guidelines and you did all of the project iteration in blue level and all red level as well . GOOD JOB GUYS!!!